blockchain/types/accountkey/account_key_weighted_multi_sig.go

```diff
@@ -138,7 +160,13 @@ func (a *AccountKeyWeightedMultiSig) SigValidationGas(currentBlockNumber uint64,
 		logger.Error("should not happen! numKeys is equal to zero!")
 		return 0, kerrors.ErrZeroLength
 	}
-	return (numKeys - 1) * params.TxValidationGasPerKey, nil
+
+	isIstanbul := fork.Rules(new(big.Int).SetUint64(currentBlockNumber)).IsIstanbul
+	if isIstanbul {
+		return uint64(sigNum-1) * params.TxValidationGasPerKey, nil
+	} else {
```

Comment:
This is minor..
I think "else" is unnecessary in here.

Reply:
reflected at https://github.com/klaytn/klaytn/pull/1001/commits/21c4c774ce713c95966cc966e39b487d0a7abd5c